doc/requirements.txt

```diff
@@ -33,3 +33,5 @@ requests~=2.28.1
 # navbar and header data to be updated at the source
 pennylane-sphinx-theme
 tomli~=2.0.0 # Drop once minimum Python version is 3.11
+sphinxext-opengraph==0.6.3 # Note: latest version 0.9.0 requires Sphinx >=4.0
```

Comment:
**Question:** Is there something holding us back from upgrading to Sphinx v4?

Reply:
No idea, I just didn't want to tackle that in this PR 😆